seismic/traveltime/conda_env_setup.sh

```diff
@@ -0,0 +1,66 @@
+# Created by Fei Zhang on 2019-12-13 in VDI
+
+# Step-1 set up install ellip-corr
+conda create -n hiperseispy37 python=3.7
```

Comment:
On home page, we only display badges for Python 3.5 and 3.6 compatibility. We should either add Python 3.7 badge, and add it to the TravisCI testing, or else drop this back to Python 3.6.

Reply:
Do this later